File: code/java/src/oopSolidSample/domain/Card.java
package oopSolidSample.domain;

public class Card
{
    public Rank Rank;
    public Suit Suit;

    public Card(Rank rank2, Suit suit2) {
		Rank = rank2;
		Suit = suit2;
	}
	public String getDescription()
    {
		return Rank + " of " + Suit;
    }
}

File: code/c-sharp/x64/src/OopSolidSample/Domain/Card.cs
using System;

namespace OopSolidSample.Domain
{
    public class Card
    {
        public Rank Rank { get; set; }
        public Suit Suit { get; set; }

        public string Description
        {
            get { return string.Format("{0} of {1}", Rank, Suit); }
        }
    }
}